src/main/java/qupath/ext/instanseg/ui/InputChannelItem.java

```diff
@@ -41,7 +41,7 @@ class InputChannelItem {
     InputChannelItem(ColorDeconvolutionStains stains, int i) {
         this.name = stains.getStain(i).getName();
         this.transform = ColorTransforms.createColorDeconvolvedChannel(stains, i);
-        this.constructor = String.format("ColorTransforms.createColorDeconvolvedChannel(stains, %d)", i);
+        this.constructor = String.format("ColorTransforms.createColorDeconvolvedChannel(QP.getCurrentImageData().getColorDeconvolutionStains(), %d)", i);
```

Comment:
Can you drop `QP` under the assumption it'll be a default import?

Reply:
Oh thanks yeah, I always get this wrong and curse the error